node/components.go

```diff
@@ -24,6 +25,7 @@ func bridgeComponents(cfg *Config, store Store) fxutil.Option {
 	return fxutil.Options(
 		fxutil.Supply(Bridge),
 		baseComponents(cfg, store),
+		fxutil.Provide(services.FullAvailability),
```

Comment:
We don't need it for Bridge and only for Full. To do this we need to remove Availability from ShareService.